spec/lib/data_update_scripts/migrate_logo_svg_data_spec.rb

```diff
@@ -1,42 +0,0 @@
-require "rails_helper"
-require Rails.root.join(
-  "lib/data_update_scripts/20220105112823_migrate_logo_svg_data.rb",
-)
-
-describe DataUpdateScripts::MigrateLogoSvgData do
```

Comment:
Should we remove the actual DUS as well since we've removed the spec?  I'm not sure if we usually go back and remove Data Update Scripts, in this case I think it makes sense. However, just want to double check with @citizen428.

Reply:
I was going to ask you about this today, @Ridhwana! We can remove the DUS using a DUS, but I'm not sure if that's something I should do in this PR.

Reply:
This is a discussion we never fully resolved. In the past, we did indeed go back and remove DUS, often just via a `git rm`. Later on, we got a bit more sophisticated and wrote DUS to remove older DUS.

![I heard you like meme](https://i.imgflip.com/63r6hz.jpg)

However, all of this got complicated quite a bit by us releasing our self-hosting solution. For DEV and the fleet, we can tell if the scripts ran everywhere, so removing isn't an issue. But we can't be sure about self-hosters. If they don't frequently update we could run into situations where a new feature was deployed that necessitated a DUS but by the time a user updates their Forem instance the actual data update script is gone already. This leaves self hosters a bit stranded, as new code will assume data in a certain format that is not reflected in the DB.

Given that DUS should be idempotent, there should be no harm in leaving them in the codebase.

Reply:
Hmm this makes sense. Thanks Michael!

 @juliannatetreault based on the above, let's not try and remove the DUS as it is idempotent based on 
```
      return if ::Settings::General.try(:logo_svg).blank? ||
        (::Settings::General.try(:original_logo).present? && ::Settings::General.try(:resized_logo).present?)
```
You can probably revert the changes that remove the spec and the uploader file too for now.

Reply:
> If we're keeping around the DUS then let's revert the changes that remove the spec and the uploader file too for now.

@Ridhwana - I have added back the `logo_svg_uploader.rb`, though I wonder if we need it since still since `logo_svg` has effectively been removed--what purpose would this uploader serve now? Additionally, while I can add back the `migrate_logo_svg_data_spec.rb`, the issue with keeping it around is that it appears to fail since `logo_svg` has been removed:

```
 1) DataUpdateScripts::MigrateLogoSvgData when the svg is able to be processed returns the logo url with a png extension
     Failure/Error: allow(Settings::General).to receive(:logo_svg).and_return(processable_svg_string)
       Settings::General(id: integer, created_at: datetime, updated_at: datetime, value: text, var: string) does not implement: logo_svg
     # ./spec/lib/data_update_scripts/migrate_logo_svg_data_spec.rb:18:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:145:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:145:in `block (2 levels) in <top (required)>'
     # ./spec/rails_helper.rb:100:in `block (2 levels) in <top (required)>'

  2) DataUpdateScripts::MigrateLogoSvgData when the svg is unable to be processed logs an error
     Failure/Error: allow(Settings::General).to receive(:logo_svg).and_return(unprocessable_svg_string)
       Settings::General(id: integer, created_at: datetime, updated_at: datetime, value: text, var: string) does not implement: logo_svg
     # ./spec/lib/data_update_scripts/migrate_logo_svg_data_spec.rb:32:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:145:in `block (3 levels) in <top (required)>'
     # ./spec/rails_helper.rb:145:in `block (2 levels) in <top (required)>'
     # ./spec/rails_helper.rb:100:in `block (2 levels) in <top (required)>'

Finished in 0.27226 seconds (files took 7.31 seconds to load)
2 examples, 2 failures
```